Tests/UI/CardViewControllerTests.swift

```diff
@@ -33,6 +33,7 @@ class CardViewControllerTests: XCTestCase {
         let cardViewController = CardViewController()
         XCTAssertEqual(cardViewController.cardHolderNameState, .required)
         XCTAssertEqual(cardViewController.billingDetailsState, .required)
+        cardViewController.isNewUI = false
```

Comment:
Can you add a test when `cardViewController.isNewUI = true `as well ?

Reply:
It's removed for the other PR 
https://github.com/checkout/frames-ios/pull/179